tools/topology/topology2/include/common/tokens.conf

```diff
@@ -145,4 +145,9 @@ Object.Base.VendorToken {
 	"sof_tkn_copier" {
 		node_type		1980
 	}
+
+	"sof_tkn_queue" {
```

Comment:
@aiChaoSONG I think we should make these part of the sof_comp_* tokens and set the default in widget-common.conf as well.